lib/Backend/Lower.cpp

```diff
@@ -8824,7 +8851,16 @@ Lowerer::LowerLdArrViewElem(IR::Instr * instr)
         }
         done = instr;
     }
-    InsertMove(dst, src1, done);
+    if (dst->IsInt64())
```

Comment:
I agree, the reason I had to do this is because we need an instance of Lowerer in order to lower int64 assign, ChangeToAssign and InsertMove are static.
I tried changing them to be methods, but if I recall there are place in LinearScan where we call these methods to handle move instructions without a Lowerer.